packages/migrations/src/Component/Doctrine/Migrations/Configuration.php

```diff
@@ -10,6 +10,10 @@
 use Doctrine\DBAL\Migrations\Version;
 use Shopsys\MigrationBundle\Component\Doctrine\Migrations\Exception\MethodIsNotAllowedException;
 
+/**
```

Comment:
TBH? I am a little bit confused with this comment - could you please remove it? The first line would make sense in the commit message, the second line seems useless to me

Reply:
this should help developers with the future development when they will not be mistaken by wrong autosuggestion in php-IDE